assets/src/block-editor/index.js

```diff
@@ -1,14 +1,9 @@
 // On running npm run dev, this will compile to assets/js/.
 
 /**
- * WordPress dependencies
+ * Internal dependencies
  */
-import { registerBlockType } from '@wordpress/blocks';
+import { registerBlocks } from './helpers';
 
-const blocks = require.context( './blocks', true, /(?<!test\/)index\.js$/ );
```

Comment:
`registerBlocks` is moved to a helpers file because I couldn't figure out how to mock `require.context()`, a Webpack function. 

But maybe moving this into a separate file isn't what you had in mind.